### PR TITLE
Cicd pipleine

### PR DESCRIPTION
**Pull request by AbijithS-aot**

### CODE REVIEW

File: .github/workflows/forms-flow-web-components-cd.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  build:
+    name: Build and push artifacts to s3
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14.17.0'
+          #TODO: force installing due to dependency conflict need to resolve the dependency
+      - run: npm ci --force
```
Comment (sreehari-aot) [inline .github/workflows/forms-flow-web-components-cd.yml:17]: is force install necessary?

**Review by kiran-aot — APPROVED** [review]

**Comment by kiran-aot** [comment]
looks good to me

File: forms-flow-webcomponent/.gitignore [hunk 2]
```diff
@@ -22,3 +22,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.parcel-cache
```
Comment (sreehari-aot) [inline forms-flow-webcomponent/.gitignore:26]: try `.parcel-cache/` a lot of unnecessary files are committed

**Review by sreehari-aot — CHANGES_REQUESTED** [review]

File: forms-flow-webcomponent/.gitignore (hunk 2 above)
Comment (AbijithS-aot) [reply forms-flow-webcomponent/.gitignore:26]: removes cache files

File: .github/workflows/forms-flow-web-components-cd.yml (hunk 1 above)
Comment (AbijithS-aot) [reply .github/workflows/forms-flow-web-components-cd.yml]: force installation not necessary removes

File: .github/workflows/forms-flow-webcomponents-ci.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  # linting:
+  #   needs: setup-job
+  #   runs-on: ubuntu-20.04
+
+  #   strategy:
+  #     matrix:
+  #       node-version: [16]
+
+  #   steps:
+  #     - uses: actions/checkout@v2
```
Comment (sreehari-aot) [inline .github/workflows/forms-flow-webcomponents-ci.yml:31]: remove unused code, keep it if its temporary comment only

**Review by sreehari-aot — APPROVED** [review]
LGTM

File: .github/workflows/forms-flow-webcomponents-ci.yml (hunk 3 above)
Comment (AbijithS-aot) [reply .github/workflows/forms-flow-webcomponents-ci.yml:31]: its temporary

**Pull request merged**